print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
```

Comment:
Can you explain to me what the difference between setNext and setValue will be in the overall code?